src/lib/eina/meson.build

```diff
@@ -199,6 +200,7 @@ sources = [
 
 if sys_windows == true
   sources += 'eina_file_win32.c'
+  sources += '_eina_fnmatch.c'
```

Comment:
I know we talked about this earlier, but anyway: you should call it `eina_fnmatch.c` :) The `_` is just for private things in source-code, not filenames.